.github/workflows/rust.yml

```diff
@@ -112,9 +112,9 @@ jobs:
           toolchain: nightly
           override: true
       - uses: Swatinem/rust-cache@v1
-      - uses: actions-rs/install@v0.1
+      - name: Install cargo-udeps, and cache the binary
+        uses: baptiste0928/cargo-install@v1
         with:
           crate: cargo-udeps
-          version: latest
```

Comment:
Just remembering that we should probably install these binaries in CI using `--locked`